ls8/cpu.c

```diff
@@ -110,6 +114,12 @@ void cpu_run(struct cpu *cpu)
 
   while (running)
   {
+  printf("\npc %03u, registers: [ ", cpu->pc);
```

Comment:
not to toot my own horn, but: yes, great idea, this'll make it 10x easier to debug